src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,377 @@
+future, though, let's break this one down a bit.
+
+The concept of Minecraft modding has existed almost as long as Minecraft itself has -- with a very diverse range of
+mods, mod-loaders and communities being created and finding their niches. That said, one mod-loader ended up taking
+the crown for a very long time -- Forge.
+
+As Forge's popularity grew and its range of mods expanded, many users became used to their workflow: they'd install
+Forge, find some mods and, since Forge was the only really viable mod-loader for a long time, those mods probably
+ just worked. Later on, other mod-loaders started appearing and disappearing. Liteloader showed 
+up with its client-side modding framework. Rift appeared, and then died. Fabric appeared, and started gaining 
+momentum -- and that's when people _really_ started getting upset.
+
+Since then, there's been a lot of complaints about additional mod-loaders "fracturing the community". We don't have
+a crystal ball, but of course the concept has some merit: as more mod-loaders pop up and more mods appear, it's
+inevitable that users are going to run into mods that they want to play with, but that are incompatible with their 
+favourite mod-loader. This can certainly be annoying, but it doesn't merit the harassment 
+that many mod developers have gotten for refusing to bring their mods to a different mod-loader. We also believe in
+the benefits that come with providing users with more choice -- for example, motivating every project to do better, or 
+the extra room for technical and modding innovation.
+
+---
+
+Regardless, it's difficult to say what precisely will happen. As is always the case with a project like this, the
+earlier days of Quilt's life will most likely cause some friction in the modding sphere, as mod developers decide
+whether they wish to support Quilt. This isn't one of Quilt's goals, nor is it our goal to try to eclipse other 
+mod-loaders and force them into obscurity. Quilt is ultimately just trying to do its best in a crowded market, and that 
+has both advantages and drawbacks.
+
+Ultimately, though, we believe that the benefits of user choice outweigh the drawbacks of a shakeup of the modding
+sphere. One of the reasons we decided to support Fabric mods on Quilt going forward was to try to help soften the
+negative impact of its release -- but we've also been working with Forge on various issues, and we've been clear in our 
+support for projects that attempt to make it easier for developers to create mods that work across mod-loaders.
+
+That said, there's only so much we can do -- developers have the right to develop mods for the mod-loaders that they
+choose, and we'd never wish to take that away from them. For most people, modding is a hobby -- there will never be a
+good reason to deny developers more choice, and an understanding of this issue is not an excuse to write off 
```

Comment:
huh? This part of the sentence doesn't make sense to me--how does understanding that there's no good reason to deny developers more choice lead to writing off developer's hard work? Probably needs re-worded.